guides/common/modules/con_managing-dhcp-using-smart-proxies.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Managing_DHCP_Using_Smart_Proxies_{context}"]
+= Managing DHCP Using {SmartProxies}
+
+{Project} can integrate with a DHCP service using {SmartProxies}.
```

Comment:
Sorry to say this, but I'd prefer it the other way around - the singular incl. in the title, ID and filename.

Reply:
It was the inconsistency that bothered me. Singular is fine too.

Reply:
Changed title, anchor, and file name to singular for both dhcp and dns.